lib/DockPanel.js

```diff
@@ -103,6 +105,54 @@ class DockPanel extends react_1.default.PureComponent {
         this.onPanelCornerDragBR = (e) => {
             this.onPanelCornerDrag(e, 'br');
         };
+        this.onPanelEdgeStartDragR = (e) => this.onPanelEdgeDragStart(e, "r");
+        this.onPanelEdgeStartDragB = (e) => this.onPanelEdgeDragStart(e, "b");
+        this.onPanelEdgeStartDragL = (e) => this.onPanelEdgeDragStart(e, "l");
+        this.onPanelEdgeStartDragT = (e) => this.onPanelEdgeDragStart(e, "t");
+        this.onPanelEdgeDragMove = (e) => {
+            let { panelData } = this.props;
+            if (e.screenX <= 0 || e.screenY <= 0)
+                return; // no idea why this happens, but eh
+            let dx = e.screenX - this._initX;
+            let dy = e.screenY - this._initY;
+            if (this._movingEdge == 't') {
```

Comment:
Please use strict equality operator ===

Reply:
I could, but my impression is that rinick will re-implement this anyway with their own code. 
Please advise if we will tweak and pull in this PR or have rinick code.